tests/functional_tests/functional_test_files/covalent/covalent.conf

```diff
@@ -0,0 +1,22 @@
+[sdk]
```

Comment:
Just curious, what was the reasoning behind choosing the functional tests directory as the home for these tests?

Reply:
I moved it (and the sample config file) to covalent_tests/shared_files/